cypress/e2e/po/lists/account-api-keys-list.po.ts

```diff
@@ -0,0 +1,21 @@
+import BaseResourceList from '~/cypress/e2e/po/lists/base-resource-list.po';
+
+/**
+ * List component for api key resources
+ */
+export default class ApiKeysListPo extends BaseResourceList {
+  actionMenu(accessKey: string) {
+    return this.resourceTable().sortableTable().rowActionMenuOpen(accessKey, 7);
+  }
+
+  details(accessKey: string, index: number) {
+    return this.resourceTable().sortableTable().rowWithName(accessKey).column(index);
+  }
+
+  /**
+   * Delete button (displays on page after row element selected)
+   */
+  deleteButton() {
```

Comment:
This delete button is a generic component of one the sortable table, would be goood to move it down to that PO in the ```  //
  // sortable-table-header
  //``` section

Reply:
makes sense. updated